pkg/kapp/cmd/app/deploy_flags.go

```diff
@@ -48,4 +49,5 @@ func (s *DeployFlags) Set(cmd *cobra.Command) {
 
 	cmd.Flags().BoolVar(&s.Logs, "logs", true, fmt.Sprintf("Show logs from Pods annotated as '%s'", deployLogsAnnKey))
 	cmd.Flags().BoolVar(&s.LogsAll, "logs-all", false, "Show logs from all Pods")
+	cmd.Flags().StringVar(&s.AppMetadataFile, "app-metadata-file", "", "Set filename to write app metadata")
```

Comment:
@praveenrewar @100mik is the plan here would also be to read from such file in future? if not, we should probably follow carvel convention of having `-output` suffix.

Reply:
sgtm; added `-output` suffix to the flag name